tests/classes/type_hinting_005d.phpt

```diff
@@ -4,11 +4,9 @@ Check type hint compatibility in overrides with array hints.
 <?php
 Class C { function f($a) {} }
 
-echo "Array hint, should be nothing.\n";
+// Array hint, should be nothing.
 Class D extends C { function f(array $a) {} }
 ?>
 ==DONE==
 --EXPECTF--
-Warning: Declaration of D::f(array $a) should be compatible with C::f($a) in %s on line 5
-Array hint, should be nothing.
-==DONE==
```

Comment:
The "==DONE==" looks like it as removed from expected text, but not from the program.